### PR TITLE
Added CMakeLists for building project with CMake

### PR DESCRIPTION
**Pull request by BondarenkoArtur**
Added CMakeLists for building project with CMake
Also fixed build errors

### CODE REVIEW

File: Show me image/Fly.cpp [hunk 1]
```diff
@@ -68,7 +68,8 @@ void	Fly::Render() {
 	float _x, _y;
 	float _dx, _dy;
 
-	const float H = 64.0f, const W = 64.0f;
+	const float H = 64.0f;
```
Comment (pavart) [inline Show me image/Fly.cpp]: Is it related to CMake?

File: Show me image/routine.cpp [hunk 2]
```diff
@@ -139,7 +139,7 @@ GLuint loadTexture (char * file,
 bool load_files() {
     image	= load_image( "images/hello.png" );
 	flyImg	= load_image( "images/fly.png" );
-	kover	= load_image( "images/KOVER.jpg" );
+	kover	= load_image( "images/KOVER.JPG" );
```
Comment (pavart) [inline Show me image/routine.cpp]: Is it also related to CMake?

**Comment by BondarenkoArtur** [comment]
I would really appreciate if someone else will try this on Linux.

**Comment by pavart** [comment]
@BondarenkoArtur I can try it later today on Mac

File: README.md [hunk 3]
```diff
@@ -21,3 +21,13 @@ But you better not press other buttons because some of them are crashing client
 
 #### Warning for explorers
 Do not expect to see a well written code here.
+
+#### To build with CMake
+
+```
+mkdir build & cd build
```
Comment (pavart) [inline README.md]: Please change to
```mkdir build && cd build```

File: CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.14)
```
Comment (pavart) [inline CMakeLists.txt]: I'm not familiar with CMake, but is it OK to set MINIMUM version of CMake to be 3.14 (which is currently latest CMake) ?
By doing this we are forcing our contributors to update CMake to the latest version.
Do we use any features from this particular version?

Comment (pavart) [reply CMakeLists.txt]: Here's some links:
https://packages.ubuntu.com/xenial/cmake
https://packages.ubuntu.com/bionic/cmake
https://packages.ubuntu.com/cosmic/cmake
As you can see, official packages of CMake for Ubuntu have lower version.

Comment (Badrpas) [reply CMakeLists.txt]: Considering that Travis CI has Ubuntu 16.04 as their latest supported platform it is wise to orient around it.
That being said - let's set it to `3.5.1` CMake.

File: Client/CMakeLists.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(Fly-simulator-client)
+
+file(GLOB srcs
```
Comment (Badrpas) [inline Client/CMakeLists.txt]: I think it's common practice to have such identifier to be called as `src`.

File: Server/CMakeLists.txt [hunk 6]
```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(Fly-simulator-server)
+
+file(GLOB srcs
```
Comment (Badrpas) [inline Server/CMakeLists.txt]: Same as in `Client/CMakeLists.txt`

File: Show me image/CMakeLists.txt [hunk 7]
```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(Fly-simulator-show-me-image)
+
+file(GLOB srcs
```
Comment (Badrpas) [inline Show me image/CMakeLists.txt]: Same as in `Client/CMakeLists.txt`

**Review by Badrpas — CHANGES_REQUESTED** [review]
Please split changes non-related to CMake to different check-in.

File: Show me image/Fly.cpp (hunk 1 above)
Comment (BondarenkoArtur) [reply Show me image/Fly.cpp]: I wasn't able to compile without this change

File: Show me image/routine.cpp (hunk 2 above)
Comment (BondarenkoArtur) [reply Show me image/routine.cpp]: In Linux file-system is case-sensitive, so we can rename files or change names into a code.

File: CMakeLists.txt (hunk 4 above)
Comment (BondarenkoArtur) [reply CMakeLists.txt]: `3.5.1` or just `3.5` ?

Comment (Badrpas) [reply CMakeLists.txt]: `3.5` should be fine

**Review by Badrpas — APPROVED** [review]

**Pull request merged**